main.py

```diff
@@ -23,10 +23,10 @@ def handle_arguments():
                         help='number of days for validation, default: 28')
     parser.add_argument('-s', '--num_models', type=int, default=500,
                         help='number of submodels to make, default: 500')
-    parser.add_argument('-d', '--dependent', action='store_true',
-                        help='switch to have inter-group dependencies')
-    parser.add_argument('-e', '--epochs', type=int, default=50, help='number '
-                        'of iterations over training data, default: 50')
+    parser.add_argument('-d', '--dropout', type=float, default=1.0,
+                        help='prob to drop inter-group weight grad, default: 1')
+    parser.add_argument('-e', '--epochs', type=int, default=50,
+                        help='number of iters over training data, default: 50')
```

Comment:
isn't it confusing to set epochs to control iters

Reply:
I had to fit within 80 columns. It is probably better with 'iterations over training data, default: 50'